python/glow/wgr/tests/test_pivot_for_gwas.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pyspark.sql import Row
+from pyspark.sql.functions import *
+from pyspark.sql.utils import AnalysisException
+import glow
+from glow.wgr import functions
+
+
+def test_error_too_many_levels(spark):
+    df = pd.DataFrame(columns=['c1', 'c2', 'c3'])
+    df = df.set_index(['c1', 'c2', 'c3'])
+    with pytest.raises(ValueError):
+        functions.pivot_for_gwas(spark, df)
+
+
+def test_pivot_loco(spark):
+    df = pd.read_csv('test-data/wgr/ridge-regression/level1YHatLoco.csv')
+    df = df.set_index(['sample_id', 'contigName'])
+    pivoted = functions.pivot_for_gwas(spark, df)
+    assert pivoted.count() == 12
+    assert pivoted.columns == ['label', 'contigName', 'values']
+    pt_list = pivoted.where('label = "sim100" and contigName = 1').head().values
+    assert len(pt_list) == 100
+    expected = [
```

Comment:
Can you add a check that the list ordering matches the input sample ID order? This would also allow us to avoid hard-coding these values.